src/System.Management.Automation/resources/Modules.resx

```diff
@@ -627,4 +627,46 @@
   <data name="CannotFindCoreCompatibleBuiltInModule" xml:space="preserve">
     <value>Cannot find the built-in module '{0}' that is compatible with the 'Core' edition. Please make sure the PowerShell built-in modules are available. They usually come with the PowerShell package under the $PSHOME module path, and are required for PowerShell to function properly.</value>
   </data>
+  <data name="WDACExportModuleCommandLogTitle" xml:space="preserve">
+    <value>Export-ModuleMember Cmdlet</value>
+  </data>
+  <data name="WDACExportModuleCommandLogMessage" xml:space="preserve">
+    <value>Export of module members will fail in Constrained Language mode because module '{0}', has a language mode '{1}' that is different from the current session '{2}'.</value>
+  </data>
+  <data name="WDACImplicitFunctionExportLogTitle" xml:space="preserve">
+    <value>Module Implicit Function Export</value>
+  </data>
+  <data name="WDACImplicitFunctionExportLogMessage" xml:space="preserve">
+    <value>Implicit function export for module '{0}' will be denied because it is trusted but the session is in ConstrainedLanguage mode (not trusted). Always export module functions individually by full name.</value>
```

Comment:
Is it enough to just say that it is not allowed in ConstrainedLanguage mode?
```suggestion
    <value>Implicit function export for module '{0}' will be denied because the session is in ConstrainedLanguage mode. Always export module functions individually by full name.</value>
```

Reply:
Updated comment to clarify.